src/README.md

```diff
@@ -196,6 +196,55 @@ int main(int argc, char* argv[]) {
 }
 ```
 
+### Performance Metrics
+
+`ov.genai.PerfMetrics` (referred to as `PerfMetrics` for simplicity) is a structure that holds performance metrics for each generate call. `PerfMetrics` hold fields with mean and standard deviations for the following metrics:
+- `ttft`
+- `tpot`
+- `load_time`
+- `generate_duration`
+- `tokenization_duration`
+- `detokenization_duration`
+- `throughput`
+
+and:
+- `num_generated_tokens`
+- `num_input_tokens`
```

Comment:
They are left because one corresponds to the number of tokens in the initial input promt and tokenization time, other to the number of generated tokens and detokenization time.

If we need them both we always can sum them. But if is do the sum on our side if someone want to separate them they will have to do extra efforts, e.g. trying to tokenize input prompt one again. I think it's better to leave separate